.github/ISSUE_TEMPLATE/other.yml

```
@@ -1,6 +1,6 @@
 name: Issue
 description: General Issue
-title: ''
+title: '[ISSUE]: '
```

Could we use a nonredundant prefix e.g. MISC for miscellaneous? What do you think

```suggestion
title: '[MISC]: '
```

---

Sure we can :) I went maybe a bit overboard with keeping it in line with `[Bug]: ` and `[FEATURE]: ` ;)